src/devices/Ssd13xx/samples/Ssd13xx.Samples.csproj

```diff
@@ -14,7 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="$(MSBuildThisFileDirectory)\..\Ssd1306.csproj" />
+    <ProjectReference Include="../Ssd13xx.csproj" />
```

Comment:
nit: `/` -> `\`

Reply:
Actually forward slash is fine here. Forward-slashes work in windows and Linux, and Linux sometimes has problems with backslashes, so we prefer forward slashes

Reply:
i just commented on that since we use backslashes everywhere. I haven't seen any backslashes issues on linux with msbuild